addons/base_vat/models/res_partner.py

```diff
@@ -232,6 +232,27 @@ def check_vat_al(self, vat):
             return True
         return False
 
+    __check_tin_hu_individual_re = re.compile(r'^8\d{9}$')
+    __check_tin_hu_companies_re = re.compile(r'^\d{8}-[1-5]-\d{2}$')
+
+    def check_vat_hu(self, vat):
+        """
+            Check Hungary VAT number that can be for example 'HU12345676 or 'xxxxxxxx-y-zz' or '8xxxxxxxxy'
+            - For xxxxxxxx-y-zz, 'x' can be any number, 'y' is a number between 1 and 5 depending on the person and the 'zz'
+              is used for region code.
+            - 8xxxxxxxxy, Tin number for individual, it has to start with an 8 and finish with the check digit
+        """
+        companies = self.__check_tin_hu_companies_re.match(vat)
+        if companies:
+            return True
+        individual = self.__check_tin_hu_individual_re.match(vat)
+        if individual:
+            return True
+        # Check the vat number
+        check_func = stdnum.util.get_cc_module('hu', 'vat').is_valid(vat)
+        if check_func is not None:
+            return check_func(vat)
```

Comment:
This means that it will always return `None` if the function doesn't exist, so it won't validate.
Let's assume that the function always exists, which is what I expect :shrug: